lib/mayaUsd/ufe/StagesSubject.cpp

```diff
@@ -247,7 +252,7 @@ void StagesSubject::stageChanged(
             const TfToken nameToken = changedPath.GetNameToken();
             auto          usdPrimPathStr = changedPath.GetPrimPath().GetString();
             auto ufePath = stagePath(sender) + Ufe::PathSegment(usdPrimPathStr, g_USDRtid, '/');
-            if (nameToken == UsdGeomTokens->xformOpOrder) {
+            if (isTransformChange(nameToken)) {
```

Comment:
Info-only changes had a different test from resync changes, which caused a bug during development of this branch, as we were missing a notification on undo.  Factored out the correct notification test into a function, and used in in both places.

Reply:
@ppt-adsk To be clear, the bug was due to the fact that the check for `UsdGeomXformOp::IsXformOp(nameToken)` was missing?

Reply:
Correct.